src/styles/blocks/card.scss

```
@@ -0,0 +1,98 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  font-family: Roboto, sans-serif;
+  background-color: #fff;
```

Move it to the `html` selector
```suggestion
  background-color: #fff;
```